bitnami/kubernetes-event-exporter/values.yaml

```diff
@@ -195,16 +251,27 @@ nodeAffinityPreset:
 ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
 ##
 affinity: {}
+## @param updateStrategy.type Deployment strategy type.
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## e.g:
+## updateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+updateStrategy:
+  type: RollingUpdate
 ## @param extraEnvVars Array containing extra env vars to be added to all containers
 ## For example:
 ## extraEnvVars:
 ##  - name: MY_ENV_VAR
 ##    value: env_var_value
 ##
 extraEnvVars: []
-## @param extraEnvVarsConfigMap ConfigMap containing extra env vars to be added to all containers
+## @param extraEnvVarsCM ConfigMap containing extra env vars to be added to all containers
 ##
-extraEnvVarsConfigMap: ""
+extraEnvVarsCM: ""
```

Comment:
May I ask why such changes are allowed within a minor chart bump?

Reply:
Ah.. only fixed in values.yaml and was already there as `extraEnvVarsCM` inside the deployment. 👌

Reply:
Hi @mkilchhofer, thank you for asking!

The value `extraEnvVarsConfigMap` referenced in the `values.yaml` was indeed a typo. The implemented value was `extraEnvVarsCM` as you can see in version  `1.3.2`:
https://github.com/bitnami/charts/blob/8cad18aed9966a6f0208e5ad6cee46cb217f47ab/bitnami/kubernetes-event-exporter/templates/deployment.yaml#L66-L69

So this isn't really a non-backward compatible change, but a bug fix.